azure-storage/README.md

```diff
@@ -0,0 +1,69 @@
+# Azure deployment
+This deployment includes deploying a storage account blob only under it's own resource group in canadacentral.  This is to be used for storage migration as part of the CWU Cloud Discovery project.
+
+These are instructions for linux-based machines, however provider documentation is linked below if you want to run from another platform
+
+
+## Terraform
+Install [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html)
+```
+wget https://releases.hashicorp.com/terraform/0.12.7/terraform_0.12.7_linux_amd64.zip
```

Comment:
We want to consider that our colleagues and clients will be catching up learning dept.

Let's script out these steps using the following patterns:

Can we avoid the use of binary packages and instead rely on OS specific package installers?  choco, brew, apt, yum?  choco for now, I port to brew when I get to running this

Conditional logic to handle package installers/binaries:
https://github.com/bcgov/eagle-dev-guides/blob/f6836e9a72ef07c8a4ffe754ed01c47df4456454/developer_install_prerequisites.sh#L5

Setting reboot tolerant environment variables:
https://github.com/bcgov/eagle-dev-guides/blob/f6836e9a72ef07c8a4ffe754ed01c47df4456454/developer_install_prerequisites.sh#L73

https://github.com/bcgov/eagle-dev-guides/blob/f6836e9a72ef07c8a4ffe754ed01c47df4456454/developer_install_helper.sh#L41

Reply:
Added a bash script to install Terraform and Azure CLI by package installers.  Don't have a mac so can't test the brew bits they line up with the packages available.  Terraform isn't available for yum or apt so I put in the steps to download and unzip into the home directory instead

We can't set the environment variables programatically as az login is interactive.  Also I'm not sure we want to set these to be kept in a users bash profile as these shouldn't be used on a daily basis, basically gives full access to an Azure account that can create any resources.  Should only be set when performing Terraform operations, which shouldn't be very often